tools/jsii-compliance/README.md

```diff
@@ -0,0 +1,65 @@
+# jsii compliance
+
+This directory contains scripts and resources to create and validate the jsii compliance suite.
+
+### Compliance Suite
+
+The compliance suite is defined as a collection of abstract test cases, that each language binding must implement individualy.
+Language specific exclusions may be added to each test separately, or to the entire suite.
+
+> See [suite.ts](./suite.ts).
+
+Note that if you add a compliance test to a specific language binding only, i.e without adding it to the suite definition, the build will fail. For example:
+
+```console
+Test 'REMOVEME' from golang report does not exist in the compliance suite. Please add it to the suite definition.
```

Comment:
"...or move it to the language specific tests"

Reply:
👍